lib/convert.js

```diff
@@ -167,7 +169,11 @@
         }
 
         // Write an image of format <format> to STDOUT
-        args.push(format + ":-");
+        if (format === "jpg" && useFlite && settings.useOptimalJPEG) {
```

Comment:
As I mentioned in the related PR, it seems like you should consider allowing a range of options (the range supported by FLITE) here instead of just toggling the "optimal" encoding.